# End Effector's Camera Pose

std_msgs/Header header

geometry_msgs/Quaternion quaternion

# xyz position
geometry_msgs/Vector3 position
